Solutions/ESETPROTECT/Analytic Rules/ESETWebsiteBlocked.yaml

```diff
@@ -0,0 +1,44 @@
+id: 7b84fc5b-9ffb-4e9b-945b-5d480e330b3f
+name: Website blocked by ESET
+description: |
+  'Create alert on websites blocked by ESET.'
+severity: Low
+requiredDataConnectors:
+  - connectorId: ESETPROTECT
+    dataTypes:
+      - Syslog
```

Comment:
If the detection query operates on Parser ,dataTypes should be the Parser name and not the table name. [Please refer this ](https://github.com/Azure/Azure-Sentinel/wiki/Query-Style-Guide#datatypes)for more details :